CHANGES.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+### New features
+
+* base: update motorParker module. by @guirodrigueslima in
+  https://github.com/cnpem/epics-in-docker/pull/124
+  * motorParker module define motor resolution and motor status was included.
```

Comment:
Suggestion: "support for motor resolution, motor status and reconfiguration upon reconnection added"

Rationale: we are already talking about motorParker in this item. No reason to replicate this information. I find "support for motor resolution" easier to read. Maybe it could be "motor resolution definition" as well. Moreover, the reconnection stuff is relevant to be mentioned. Finally, "added" is shorter than "were included".